### ISSUE TITLE
Clothing1M setting with Resnet50

### ISSUE THREAD

**Issue by wuterry** [NUST-Machine-Intelligence-Laboratory/Jo-SRC#1]
When I train clothing1m with resnet50 by default configures, the best accuracy is 71.89, may be the parameter is not correct. Can you provide the settings of clothing1m.cfg when training with Resnet50? Thanks.

**Comment by 71-71** [comment]
Hi, great inspired by your work. But I am curious about how to calculate the accuracy listed in the paper. Given two networks,  is the result calculated by the mean accuracy of two classifiers or use the mean mixed logits of two networks as the final representations then calculate accuracy?
